OlinkAnalyze/R/Read_NPX_data.R

```diff
@@ -1,6 +1,6 @@
 #' Function to read NPX data into long format
 #'
-#' Imports an NPX file exported from Olink Software.
+#' Imports an NPX or QUANT file exported from Olink Software.

```

Comment:
Since most of the OA functions expect NPX and will not work with quant data out of the box, do we want to help the user with a warning message when they read QUANT files?